test/unit/3dtileslayerprocessbatchtable.js

```diff
@@ -8,34 +9,58 @@ import View from 'Core/View';
 import GlobeView from 'Core/Prefab/GlobeView';
 import { HttpsProxyAgent } from 'https-proxy-agent';
 import Coordinates from 'Core/Geographic/Coordinates';
+import Fetcher from 'Provider/Fetcher';
+import sinon from 'sinon';
 import Renderer from './bootstrap';
 
+import tilesetBatchTable from '../data/3dTiles/tilesetBatchTable.json';
+
+const tileBatchTable = fs.readFileSync('./test/data/3dTiles/tile.b3dm');
```

Comment:
Use a relative path.

```suggestion
const tileBatchTable = fs.readFileSync('../data/3dTiles/tile.b3dm');
```

Reply:
I tried to do so but (at least on my computer) it's not working. It might be linked with my platform.